src/smc-util/message.coffee

```diff
@@ -448,7 +481,7 @@ message
 # starting_bundle_number.
 #
 # client --> hub --> project_server
-API message
+API message # unimplemented
```

Comment:
OK, they must have got removed from the implementation; I think they should get deleted entirely then, and that functionality is now provided in a different way (i.e., via the query message type).